addons/im_livechat/static/tests/chat_window_patch.test.js

```diff
@@ -110,3 +117,24 @@ test("Focus should not be stolen when a new livechat open", async () => {
     await animationFrame();
     await contains(".o-mail-Composer-input[placeholder='Message #general…']:focus");
 });
+
+test("do not ask confirmation if other operators are present", async () => {
+    const pyEnv = await startServer();
+    const guestId = pyEnv["mail.guest"].create({ name: "Visitor #12" });
+    const otherOperatorId = pyEnv["res.partner"].create({ name: "John" });
+    const channelId = pyEnv["discuss.channel"].create({
+        channel_member_ids: [
+            Command.create({ partner_id: serverState.partnerId }),
+            Command.create({ guest_id: guestId }),
+            Command.create({ partner_id: otherOperatorId }),
+        ],
+        livechat_operator_id: serverState.partnerId,
+        channel_type: "livechat",
+        livechat_active: true,
+    });
+    setupChatHub({ opened: [channelId] });
+    await start();
+    await contains(".o-mail-ChatWindow");
+    await click("[title*='Close Chat Window']");
+    await contains(".o_notification", { text: "You left Visitor #12." });
+});
```

Comment:
In 18.2, there's no `o_notification` from leaving. Test should be adapted, e.g. chat window is just removed.